src/widgets/window.js

```diff
@@ -258,7 +259,7 @@ var FlatsealWindow = GObject.registerClass({
             showPermissionsButton = false;
         }
 
-        const isFolded = this._headerLeaflet.folded;
+        const isFolded = this._contentLeaflet.folded;
         if (isFolded) {
             showApplicationsButton = true;
             showPermissionsButton = true;
```

Comment:
Minor issue:

![image](https://user-images.githubusercontent.com/481418/105614404-0681f080-5da8-11eb-8411-7d6c0ee7fe23.png)

For some reason, non `FlatsealPermissionSwitchRow` are a little bit bigger than they should be. Ideally, these rows should have the same height as `FlatsealPermissionSwitchRow` rows.

Reply:
![Screenshot from 2021-01-23 14-07-46](https://user-images.githubusercontent.com/29347290/105615305-69fa2700-5d84-11eb-9723-749aafe79bb3.png)
How is this?

Reply:
hard to tell by eye, maybe check with inspector? just to make sure it has the same height as the `FlatsealPermissionSwitchRow` :)

This is what I am doing currently https://github.com/tchx84/Flatseal/blob/master/src/widgets/permissionSwitchRow.ui#L11

Reply:
They do :)